x-pack/plugins/ingest_manager/common/services/setup.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { HttpSetup } from 'kibana/public';
+import { setupRouteService } from './routes';
+
+export async function setupIngestManager(http: HttpSetup) {
+  await http.post(setupRouteService.getSetupPath());
```

Comment:
Suggestion: keep a module level variable here that track if the setup was already called and if so, just return the prior response. I think we only need to do this once, right?